kategory/src/test/kotlin/kategory/laws/MonadStateLaws.kt

```diff
@@ -0,0 +1,36 @@
+package kategory
+
+import io.kotlintest.properties.forAll
+
+object MonadStateLaws {
+
+    inline fun <reified F> laws(M: MonadState<F, Int> = monadState<F, Int>(), EQ: Eq<HK<F, Int>>): List<Law> =
+            MonadLaws.laws(M, EQ) + listOf(
+                    Law("Monad State Laws: idempotence", { monadStateGetIdempotent(M, Eq.any()) }),
+                    Law("Monad State Laws: set twice", { monadStateSetTwice(M, Eq.any()) }),
+                    Law("Monad State Laws: set get", { monadStateSetTwice(M, Eq.any()) }),
+                    Law("Monad State Laws: get set", { monadStateSetTwice(M, Eq.any()) })
```

Comment:
Nono, it's not a typo. I was just ignoring the EQ already passed as argument to compare equality on 2 StateTKind<Try.F, Int, Int> and just sending Eq.any() for all the laws internally. That's why they don't pass. They also need different implementations of EQ though, the one passed as an argument is not under the required constrains.